administrator/templates/atum/scss/blocks/_sidebar.scss

```
@@ -520,8 +520,8 @@
     [dir=rtl] & {
       right: auto;
       left: 10px;
-    }
-
+    } 
```

```suggestion
    }
```
Please remove the trailing space after the closing curly bracket here.  You can just click the button to accept my suggested change to do so.

---

Done sir please review again